src/main/kotlin/lotto/model/Lottos.kt

```diff
@@ -0,0 +1,21 @@
+package lotto.model
+
+class Lottos(
+    private val lottos: List<Lotto>,
+) {
+    fun matchlottos(
```

Comment:
```suggestion
    fun matchLottos(
```
camel case 잘 지켜주세요~!